diagrams/toolbox/additional_artifact_types.yaml

```diff
@@ -0,0 +1,26 @@
+######################################################################
```

Comment:
I would rename this folder to `.cloudnet-tosca-toolbox` for being clearer. It's important that this directory name starts with the dot (`.`) to keep it hidden. This is needed because xOpera examples are already separated into folders and we cannot have a folder for generating diagrams on the same level.